.github/workflows/tests.yml

```diff
@@ -37,19 +39,23 @@ jobs:
         key: ${{ runner.os }}-${{ github.sha }}-jlm
     - name: "Install LLVM and Clang"
       uses: ./.github/actions/InstallLlvmDependencies
+    - name: Configure jlm
+      run: CXX=clang++-16 ./configure.sh release
     - name: Compile jlm
-      run: make jlm-release -j `nproc`
+      run: make -O -j `nproc`
     - name: Run unit and C tests
-      run: make check -j `nproc`
+      run: make check -O -j `nproc`
 
   gcc:
```

Comment:
clang used to be the default compiler, and this test was to assure that there were no errors compiling with g++.
If g++ is used as the default compiler, then this test should be changed to run with clang.